installer/templates/phx_single/lib/app_name/application.ex

```diff
@@ -15,7 +15,9 @@ defmodule <%= @app_module %>.Application do
       # Start the PubSub system
       {Phoenix.PubSub, name: <%= @app_module %>.PubSub},
       # Start the Endpoint (http/https)
-      <%= @endpoint_module %>
+      <%= @endpoint_module %>,<%= if @mailer do %>
+      # Start Finch for Swoosh
+      {Finch, name: Swoosh.Finch}<% end %>
```

Comment:
I would go with `PubSub` example and set the `name` option to: `<%= @app_module %>.Finch` instead.

Reply:
Sure, happy to make the change, but `Swoosh.Finch` is expected by `Swoosh.ApiClient.Finch`. We can certainly add a config to address that.

https://github.com/swoosh/swoosh/blob/78f07b5dd5fa5af1cccb1619f103cc4b611c3cb6/lib/swoosh/api_client/finch.ex#L69

Reply:
@christopherlai we should use a configuration, yes! :)

Reply:
Btw, we should start this before the endpoint, otherwise we can try to send emails before Finch is ready. :)

Reply:
@Eiji7 @josevalim addressed in [cacf1b5](https://github.com/phoenixframework/phoenix/pull/5052/commits/cacf1b5523d09327aa5469b393818b68facf25cd)